From faa17f09218ed7e2ce4362cc2d9319f8d5b7a37f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 28 May 2014 14:52:03 +0200
Subject: [PATCH] kernel-modules/installkernel:
 s/blk_init_queue/blk_cleanup_queue/g

using "blk_cleanup_queue" will pickup also the nvme driver

https://bugzilla.redhat.com/show_bug.cgi?id=1041484
---
 modules.d/90kernel-modules/installkernel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90kernel-modules/installkernel b/modules.d/90kernel-modules/installkernel
index 04ff299..c8f948e 100755
--- a/modules.d/90kernel-modules/installkernel
+++ b/modules.d/90kernel-modules/installkernel
@@ -1,7 +1,7 @@
 #!/bin/bash
 if [[ -z $drivers ]]; then
     block_module_test() {
-	local blockfuncs='ata_scsi_ioctl|scsi_add_host|blk_init_queue|register_mtd_blktrans|scsi_esp_register|register_virtio_device'
+	local blockfuncs='ata_scsi_ioctl|scsi_add_host|blk_cleanup_queue|register_mtd_blktrans|scsi_esp_register|register_virtio_device'
 
 	egrep -q "$blockfuncs" "$1"
     }
